src/components/Activities/Activity3/Game/ABTestingReport/index.tsx

```diff
@@ -55,7 +61,9 @@ function ABTestingReport(): JSX.Element {
       </div>
     </div>
     <div>
-      <button className="playnet-button playnet-btn-blue" disabled={daysLeft <= 0} onClick={() => setState(A3_GAME_STATE.PriorityWeighing)}>Go back to variables</button>
+      <button className="playnet-button playnet-btn-blue" disabled={daysLeft < LOW_DAY_THRESHOLD} onClick={retry}>
```

Comment:
nit: i think the `LOW_DAY_THRESHOLD` just happens to be 14 and should probably be sum of `DEFAULT_TIME_ALLOCATION` instead?
```suggestion
      <button className="playnet-button playnet-btn-blue" disabled={daysLeft < objectSum(DEFAULT_TIME_ALLOCATION)} onClick={retry}>
```